cohortextractor/patients.py

```diff
@@ -383,7 +384,7 @@ def mean_recorded_value(
         This creates a variable `bp_sys` returning a float of the most recent systolic blood pressure from
         the record within the time period. In the event of repeated measurements on the same day, these
         are averaged. Patient who do not have this information
-        available do not return a value:
+        available do not return a value.  The date of measurement is returned as `bp_sys_date_measured`:
```

Comment:
We may also wish to note that the date of measurement is to year-month resolution.